hazelcast/src/test/java/com/hazelcast/map/impl/MapStatisticsAwareServiceTest.java

```diff
@@ -43,7 +43,7 @@ public void getStats_whenMapHasData() {
         HazelcastInstance hz = createHazelcastInstance();
         warmUpPartitions(hz);
         // when one map exists
-        hz.getMap("map").put(1, 1);
+        hz.getMap("map");
```

Comment:
maybe keep this test as is to match its goal (`getStats_whenMapHasData`) and add a separate to test the case where proxy is created but no data have been stored yet?